### PR TITLE
Do not set interpolation for nominal color scales

### LINKED ISSUE #4775: Do not set interpolation for nominal color scales
For example in https://vega.github.io/editor/#/examples/vega-lite/stacked_bar_weather, we should not set the `interpolate` property on the color scale.

### PR DESCRIPTION
**Pull request by kanitw**
Fix #4775

### CODE REVIEW

File: src/compile/scale/properties.ts [hunk 1]
```diff
@@ -192,8 +192,8 @@ export function bins(model: Model, fieldDef: TypedFieldDef<string>, channel: Cha
   return undefined;
 }
 
-export function interpolate(channel: Channel) {
-  if (contains([COLOR, FILL, STROKE], channel)) {
+export function interpolate(channel: Channel, scaleType: ScaleType) {
+  if (contains([COLOR, FILL, STROKE], channel) && scaleType !== 'ordinal') {
```
Comment (domoritz) [inline src/compile/scale/properties.ts]: We might want interpolation for Vega-Lite `ordinal` but not `nominal` encodings. Let's see what changes.

Comment (kanitw) [reply src/compile/scale/properties.ts]: oh right

Comment (kanitw) [reply src/compile/scale/properties.ts]: Changed.

**Comment by domoritz** [comment]
Thanks. This is great.

**Pull request merged**